Signal/src/call/OWSWebRTCDataProtos.pb.m

```diff
@@ -57,7 +57,7 @@ - (BOOL) isInitialized {
 }
 - (void) writeToCodedOutputStream:(PBCodedOutputStream*) output {
   if (self.hasId) {
-    [output writeUInt64:1 value:self.id];
```

Comment:
Those file changes are reverted, sorry mate! 
If there is any other possibility to fix/remove the project warnings I would appreciate it to help, because new warnings will be seen and fixed faster 😃